corehq/apps/export/utils.py

```diff
@@ -0,0 +1,8 @@
+from .const import TRANSFORM_FUNCTIONS
+from .exceptions import ExportInvalidTransform
+
+
+def is_valid_transform(value):
+    for transform in value:
+        if transform not in TRANSFORM_FUNCTIONS:
+            raise ExportInvalidTransform('{} is not a valid transform'.format(value))
```

Comment:
Are you expecting we might want to do more validation here in the future? If not, I think it makes more sense to just do `ListProperty(choices=TRANSFORM_FUNCTIONS)`